spec/lib/whitehall_importer/integrity_checker_spec.rb

```diff
@@ -146,6 +157,28 @@ def problem_message(attribute, expected, actual)
       stub_publishing_api_has_item(publishing_api_item)
     end
 
+    it "returns a problem when first_published_at times are not similar" do
+      publishing_api_item[:details][:first_public_at] = "2020-03-11T12:00:00.000+00:00"
+      stub_publishing_api_has_item(publishing_api_item)
+
+      expect(integrity_check.problems).to include(
+        problem_message("first_published_at",
+                        publishing_api_item[:details][:first_public_at],
+                        edition.document.first_published_at.as_json),
+      )
+    end
+
+    it "returns a problem when first_published_at times are invalid" do
+      publishing_api_item[:details][:first_public_at] = "Not a time"
+      stub_publishing_api_has_item(publishing_api_item)
+
+      expect(integrity_check.problems).to include(
+        problem_message("first_published_at",
+                        publishing_api_item[:details][:first_public_at],
+                        edition.document.first_published_at.as_json),
+      )
+    end
```

Comment:
I don't think you'll need this test here - should end up being covered by tests for the static method.

Reply:
Removed!